docs/dm/zh_CN/use-cases/shard-merge.md

```diff
@@ -0,0 +1,265 @@
+分库分表合并场景
+===
+
+### 索引
+- [场景描述](#场景描述)
+   - [上游实例](#上游实例)
+   - [同步需求](#同步需求)
+   - [下游实例](#下游实例)
+- [同步方案](#同步方案)
+- [任务配置](#任务配置)
+
+### 场景描述
+
+描述一个分库分表合并的业务场景，将上游三个 MySQL 实例的分库和分表同步到下游一个 TiDB 集群中。
+
+#### 上游实例
+
+假设上游存在下面的 schema
+
+- instance 1
+
+| schma | tables|
+|------:|:------|
+| user  | information, log_north, log_bak|
+| store_01 | sale_01, slae_02 |
+| store_02 | sale_01, slae_02 |
+
+- instance 2
+
+| schma | tables|
+|------:|:------|
+| user  | information, log_east, log_bak|
+| store_01 | sale_01, slae_02 |
+| store_02 | sale_01, slae_02 |
+
+- instance 3
+
+| schma | tables|
+|------:|:------|
+| user  | information, log_south, log_bak|
+| store_01 | sale_01, slae_02 |
+| store_02 | sale_01, slae_02 |
+
+
+#### 同步需求
+
+1. 三个实例的 `user`.`information` 合并到下游 TiDB 的 `user`.`information`
+2. 三个实例的 `user`.`log_{north|south|east}` 合并到下游 TiDB 的 `user`.`log_{north|south|east}`
+3. 三个实例的 `store_{01|02}`.`sale_{01|02}` 同步到下游 TiDB 的 `store`.`sale`
+4. 过滤掉三个实例的 `user`.`log` 表的所有删除操作
+5. 过滤掉三个实例的 `user`.`information` 表的所有删除操作
+6. 过滤掉三个实例的 `store_{01|02}`.`sale_{01|02}` 表的所有删除操作
+7. 过滤掉三个实例的 `user`.`log`
+8. `store_{01|02}`.`sale_{01|02}` 表存在 bigint 类型的自增主键，合并有冲突
+
+#### 下游实例
+
+假设同步到下游后的 schema
+
+| schma | tables|
+|------:|:------|
+| user | information, log_north, log_east, log_south|
+| store | sale |
+
+### 同步方案
+
+- 同步需求 1 和 2 通过设置 table 路由来解决，配置如下
+
+```yaml
+routes:
+  ...
+  user-route-rule:
+    schema-pattern: "user"
+    target-schema: "user"
+```
+
+***
+
+- 同步需求 3 可以通过同步功能 [table 路由](../features/table-route.md) 来解决，配置如下
+
+```yaml
+routes:
+  ...
+  store-route-rule:
+    schema-pattern: "store_*"
+    target-schema: "store"
+  sale-route-rule:
+    schema-pattern: "store_*"
+    table-pattern: "sale_*"
+    target-schema: "store"
+    target-table:  "sale"
+```
+
+***
+
+- 同步需求 4 和 5 可以通过同步功能 [binlog filter](../features/binlog-filter.md) 来解决, 配置如下
+
+```yaml
+filters:
+  ...
+  user-filter-rule:
+    schema-pattern: "user"
```

Comment:
this rule will filter all tables in `user` rather than `log*` and `information` (although only these tables in it), need more description?